pypeit/ginga/PypeIt.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
+
+from ginga import GingaPlugin
+from ginga.AstroImage import AstroImage
+
+
+class PypeItImage(AstroImage):
+    """
+    Custom image type for PypeIt
+    """
+    def __init__(self, wav_np=None, **kwargs):
+
+        AstroImage.__init__(self, **kwargs)
+
+        self.wav_np = wav_np
+
+    def info_xy(self, data_x, data_y, settings):
```

Comment:
doc string

Reply:
I did my best, but Eric Ejeschke wrote this and I don't fully get how it works.